FabricObserver/Observers/Utilities/Telemetry/AppInsightsTelemetry.cs

```diff
@@ -128,12 +130,20 @@ public string Key
             {
                 cancellationToken.ThrowIfCancellationRequested();
 
-                SeverityLevel sev = (state == HealthState.Error) ? SeverityLevel.Error
+                var sev = (state == HealthState.Error) ? SeverityLevel.Error
                                     : (state == HealthState.Warning) ? SeverityLevel.Warning : SeverityLevel.Information;
 
-                var tt = new TraceTelemetry($"{applicationName}: Service Fabric Health report - {Enum.GetName(typeof(HealthState), state)} -> {source}:{property}", sev);
+                string healthInfo = string.Empty;
+
+                if (!string.IsNullOrEmpty(unhealthyEvaluations))
+                {
+                    healthInfo += "\n" + unhealthyEvaluations;
```

Comment:
Would using Environment.NewLine be better here?

Reply:
Probably. Will modify.